.github/workflows/blog-image-tests.yml

```
@@ -0,0 +1,58 @@
+      - name: Run blog image tests
+        run: pnpm test:blog-images
+
+      - name: Upload test results
+        uses: actions/upload-artifact@v4
+        if: failure()
+        with:
+          name: blog-image-test-results-${{ matrix.node-version }}
+          path: |
+            coverage/
+            test-results/
+          retention-days: 7
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)